notioncli/cli.py

```diff
@@ -0,0 +1,170 @@
+import functools
+import os
+import argparse
+import sys
+
+from termcolor import colored, cprint
+
+from notion.client import NotionClient
+from notion.block import TodoBlock
+
+from contextlib import redirect_stdout
+from contextlib import contextmanager
+from io import StringIO
+
+@contextmanager
+def capture_stdout(output): 
+    stdout = sys.stdout
+    sys.stdout = output
+    try:
+        yield
+    finally:
+        sys.stdout = stdout
+
+
+class ConfigurationError(Exception):
+    pass
+
+
+def connect():
+    try:
+        client = NotionClient(token_v2=os.environ['NOTION_TOKEN'], monitor=False)
+    except:
+        raise ConfigurationError('Invalid or expired NOTION_TOKEN')
+
+    return client
+
+
+def get_page(client):
```

Comment:
I split these up because it's plausible that you would want to do something else with the client, but that's not practically happening right now so idk a little pushing the peas around I guess.

Reply:
makes sense